src/components/Calculator/Calculator.js

```diff
@@ -74,7 +74,6 @@ export default function Calculator() {
       default:
         return;
     }
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219302435-aed94e51-117a-49b6-8284-05da0e592812.png)

- [ ] This link is not working :fearful: Kindly fix this link to your issues page to make your readme file descriptive and professional :+1: